### PR TITLE
Enrich instructions for string functions FORMAT() and FROM_BASE64()

### PR DESCRIPTION
**Pull request by ittuann**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/16085
- Other reference link(s): https://github.com/pingcap/docs/issues/15487

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: functions-and-operators/string-functions.md [hunk 1]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
 
 ### [`FORMAT()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_format)
 
-返回指定小数位数格式的数字
+`FORMAT()` 函数用于将数字四舍五入并将其格式化为指定的小数位数，返回一个按照指定小数位数格式化后的数字。
```
Comment (xzhangxian1008) [inline functions-and-operators/string-functions.md:143]: format 函数的简介可以参考 [这里](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_format:~:text=Formats%20the%20number%20X%20to%20a%20format%20like%20%27%23%2C%23%23%23%2C%23%23%23.%23%23%27%2C%20rounded%20to%20D%20decimal%20places%2C%20and%20returns%20the%20result%20as%20a%20string.%20If%20D%20is%200%2C%20the%20result%20has%20no%20decimal%20point%20or%20fractional%20part.%20If%20X%20or%20D%20is%20NULL%2C%20the%20function%20returns%20NULL.)

File: functions-and-operators/string-functions.md [hunk 2]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
...
+**行为细节：**
+
+- 如果输入的第一个参数为字符串，且该字符串中只包含数字时，该函数将按照该数字返回结果。例如，`FORMAT('12.36', 1)` 与 `FORMAT(12.36, 1)` 的返回结果相同。
+- 如果输入的第一个参数为科学计数法（`E/e`）表示的数字时，该函数将按照该数字返回结果。例如，`FORMAT('1E2', 3)`），函数返回 `100.000`。
+- 如果输入的第一个参数为非数字开头的字符串时，该函数除了返回零值外，还返回一个警告`(Code 1292)`。例如，`FORMAT('q12.36', 5)`），函数返回 `0.00000`，但会包含警告`Warning (Code 1292): Truncated incorrect DOUBLE value: 'q12.36'`。
+- 如果输入的第一个参数为数字和非数字混合的字符串时，该函数将基于该参数中开头连续的数字部分返回结果，还返回一个警告`(Code 1292)`。例如，`“FORMAT('12.36q56.78', 1)` 与 `FORMAT('12.36', 1)` 的返回的数字结果相同，但还包括一个警告`Warning (Code 1292): Truncated incorrect DOUBLE value: '12.36q56.78'`。
+- 如果输入的第二个参数为负数，函数将四舍五入小数部分并返回整数。
+- 如果输入的任意参数为 `NULL`，函数将返回 `NULL`。
+
+**示例：**
```
Comment (xzhangxian1008) [inline functions-and-operators/string-functions.md:161]: 请举一个结果中包含逗号的例子，还有包含第三方参数的例子，具体可以参考[这里](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_format:~:text=mysql%3E%20SELECT%20FORMAT(12332.123456%2C%204)%3B%0A%20%20%20%20%20%20%20%20%2D%3E%20%2712%2C332.1235%27%0Amysql%3E%20SELECT%20FORMAT(12332.1%2C4)%3B%0A%20%20%20%20%20%20%20%20%2D%3E%20%2712%2C332.1000%27%0Amysql%3E%20SELECT%20FORMAT(12332.2%2C0)%3B%0A%20%20%20%20%20%20%20%20%2D%3E%20%2712%2C332%27%0Amysql%3E%20SELECT%20FORMAT(12332.2%2C2%2C%27de_DE%27)%3B%0A%20%20%20%20%20%20%20%20%2D%3E%20%2712.332%2C20%27)

File: functions-and-operators/string-functions.md [hunk 3]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
...
+| FORMAT(12.36, 2) |
++------------------+
+| 12.36            |
++------------------+
+```
 
 ### [`FROM_BASE64()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_from-base64)
 
-解码 base-64 表示的字符串，并返回结果
+`FROM_BASE64()` 函数用于解码 [Base64](https://datatracker.ietf.org/doc/html/rfc4648) 表示的字符串，返回其十六进制形式的字符串。
```
Comment (xzhangxian1008) [inline functions-and-operators/string-functions.md:194]: 函数简介可以参考[这里](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_format:~:text=Takes%20a%20string%20encoded%20with%20the%20base%2D64%20encoded%20rules%20used%20by%20TO_BASE64()%20and%20returns%20the%20decoded%20result%20as%20a%20binary%20string.%20The%20result%20is%20NULL%20if%20the%20argument%20is%20NULL%20or%20not%20a%20valid%20base%2D64%20string.%20See%20the%20description%20of%20TO_BASE64()%20for%20details%20about%20the%20encoding%20and%20decoding%20rules.)

File: functions-and-operators/string-functions.md (hunk 2 above)
Comment (ittuann) [reply functions-and-operators/string-functions.md:160]: 我尝试了可选的第三个参数用来指定一个区域格式，但结果出现了警告

```sql
mysql> SELECT FORMAT(12332.2,2,'de_DE');
+---------------------------+
| FORMAT(12332.2,2,'de_DE') |
+---------------------------+
| 12,332.20                 |
+---------------------------+
1 row in set, 1 warning (0.00 sec)
```
```sql
+---------+------+-------------------------+
| Level   | Code | Message                 |
+---------+------+-------------------------+
| Warning | 1649 | Unknown locale: 'de_DE' |
+---------+------+-------------------------+
```

**Comment by ittuann** [comment]
👋@xzhangxian1008 感谢您的反馈和建议！我已经在最新提交中加入了这些改进~

File: functions-and-operators/string-functions.md [hunk 4]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
...
+| FORMAT(1234.56, 1, 'en_US') |
++-----------------------------+
+| 1,234.6                     |
++-----------------------------+
+```
 
 ### [`FROM_BASE64()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_from-base64)
 
-解码 base-64 表示的字符串，并返回结果
+`FROM_BASE64(str)` 函数用于将使用 [`TO_BASE64()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_to-base64) 函数遵循的 [Base64](https://datatracker.ietf.org/doc/html/rfc4648) 编码规则进行编码的字符串解码，并将解码结果返回为十六进制字符串。
```
Comment (xzhangxian1008) [inline functions-and-operators/string-functions.md:194]:
```suggestion
`FROM_BASE64(str)` 函数接收一个遵循 [Base64](https://datatracker.ietf.org/doc/html/rfc4648) 编码规则进行编码的字符串，对其进行解码后将结果以十六进制字符串的形式进行返回。
```

File: functions-and-operators/string-functions.md [hunk 5]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
...
+
+- 此函数接受一个单一参数：需要解码的 Base64 编码字符串。
+- 如果输入参数为NULL或不是有效的 Base64 编码字符串，`FROM_BASE64()` 函数将返回 `NULL`。
```
Comment (xzhangxian1008) [inline functions-and-operators/string-functions.md:197]:
```suggestion
- 如果输入参数为 `NULL` 或不是有效的 Base64 编码字符串，`FROM_BASE64()` 函数将返回 `NULL`。
```

**Review by xzhangxian1008 — APPROVED** [review]
LGTM

File: functions-and-operators/string-functions.md [hunk 6]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
 
 ### [`FORMAT()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_format)
 
-返回指定小数位数格式的数字
+`FORMAT(X,D[,locale])` 函数用于将数字X四舍五入格式化为类似于“#,###,###.##”的格式，保留D位小数，并将结果作为字符串返回。
```
Comment (hfxsd) [inline functions-and-operators/string-functions.md:143]:
```suggestion
`FORMAT(X,D[,locale])` 函数用于将数字 X 四舍五入格式化为类似于 “#,###,###.##” 的格式，保留 D 位小数，并将结果作为字符串返回。
```

File: functions-and-operators/string-functions.md [hunk 7]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
...
+
+**参数：**
```
Comment (hfxsd) [inline functions-and-operators/string-functions.md:145]:
```suggestion
#### 参数
```

File: functions-and-operators/string-functions.md [hunk 8]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
...
+`FROM_BASE64(str)` 函数接收一个遵循 [Base64](https://datatracker.ietf.org/doc/html/rfc4648) 编码规则进行编码的字符串，对其进行解码后将结果以十六进制字符串的形式进行返回。
+
+- 此函数接受一个单一参数：需要解码的 Base64 编码字符串。
+- 如果输入参数为 `NULL` 或不是有效的 Base64 编码字符串，`FROM_BASE64()` 函数将返回 `NULL`。
+
+> **注意：**
+>
+> `FROM_BASE64()` 函数仅处理合法有效的 Base64 编码字符串。
+
+**示例：**
```
Comment (hfxsd) [inline functions-and-operators/string-functions.md:203]:
```suggestion
#### 示例
```

File: functions-and-operators/string-functions.md [hunk 9]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
...
+
+1. **`X`：数字或数字字符串：** 要格式化的数字。可以是直接的数字值、数字字符串、或科学记数法格式的数字。
+2. **`D`：小数位数：** 指定返回值的小数位数。
+3. **`[locale]`：区域设置（可选参数）：** 可选的第三个参数允许指定一个区域设置，用于结果数字的小数点、千位分隔符和分隔符之间的分组。合法的区域设置值与 `lc_time_names` 系统变量的合法值相同。如果未指定或者区域设置为 `NULL`，则默认使用 `'en_US'` 区域设置。
+
+**行为细节：**
+
+- 如果输入的第一个参数为字符串，且该字符串中只包含数字时，该函数将按照该数字返回结果。例如，`FORMAT('12.36', 1)` 与 `FORMAT(12.36, 1)` 的返回结果相同。
+- 如果输入的第一个参数为科学计数法（`E/e`）表示的数字时，该函数将按照该数字返回结果。例如，`FORMAT('1E2', 3)`），函数返回 `100.000`。
+- 如果输入的第一个参数为非数字开头的字符串时，该函数除了返回零值外，还返回一个警告`(Code 1292)`。例如，`FORMAT('q12.36', 5)`），函数返回 `0.00000`，但还会包含一个警告`Warning (Code 1292): Truncated incorrect DOUBLE value: 'q12.36'`。
```
Comment (hfxsd) [inline functions-and-operators/string-functions.md:155]:
```suggestion
- 如果输入的第一个参数为非数字开头的字符串时，该函数除了返回零值外，还返回一个警告 `(Code 1292)`。例如，`FORMAT('q12.36', 5)` 函数返回 `0.00000`，还会包含一个警告 `Warning (Code 1292): Truncated incorrect DOUBLE value: 'q12.36'`。
```

File: functions-and-operators/string-functions.md [hunk 10]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
...
+| FORMAT(1234.56, 1, 'en_US') |
++-----------------------------+
+| 1,234.6                     |
++-----------------------------+
+```
 
 ### [`FROM_BASE64()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_from-base64)
 
-解码 base-64 表示的字符串，并返回结果
+`FROM_BASE64(str)` 函数接收一个遵循 [Base64](https://datatracker.ietf.org/doc/html/rfc4648) 编码规则进行编码的字符串，对其进行解码后将结果以十六进制字符串的形式进行返回。
```
Comment (hfxsd) [inline functions-and-operators/string-functions.md:194]:
```suggestion
`FROM_BASE64(str)` 函数接收一个遵循 [Base64](https://datatracker.ietf.org/doc/html/rfc4648) 编码规则进行编码的字符串，对其解码后，将结果以十六进制字符串的形式返回。
```

File: functions-and-operators/string-functions.md [hunk 11]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
...
+
+- 此函数接受一个单一参数：需要解码的 Base64 编码字符串。
+- 如果输入参数为 `NULL` 或不是有效的 Base64 编码字符串，`FROM_BASE64()` 函数将返回 `NULL`。
```
Comment (hfxsd) [inline functions-and-operators/string-functions.md:197]:
```suggestion
- 此函数接受一个单一参数，即需要解码的 Base64 编码字符串。
- 如果输入参数为 `NULL` 或无效的 Base64 编码字符串，`FROM_BASE64()` 函数将返回 `NULL`。
```

File: functions-and-operators/string-functions.md [hunk 12]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
...
+
+> **注意：**
+>
+> `FROM_BASE64()` 函数仅处理合法有效的 Base64 编码字符串。
+
+**示例：**
+
+解码 Base64 编码的字符串 `'SGVsbG8gVGlEQg=='`（该字符串是`'Hello TiDB'`经过 Base64 编码的结果）
```
Comment (hfxsd) [inline functions-and-operators/string-functions.md:205]:
```suggestion
以下示例解码 Base64 编码的字符串 `'SGVsbG8gVGlEQg=='`，该字符串是 `'Hello TiDB'` 经过 Base64 编码的结果。
```

File: functions-and-operators/string-functions.md [hunk 13]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
...
+```sql
+mysql> SELECT CONVERT(FROM_BASE64('SGVsbG8gVGlEQg==') USING utf8mb4);
++--------------------------------------------------------+
+| CONVERT(FROM_BASE64('SGVsbG8gVGlEQg==') USING utf8mb4) |
++--------------------------------------------------------+
+| Hello TiDB                                             |
++--------------------------------------------------------+
+```
+
+解码 Base64 编码的数字 `MTIzNDU2`（该字符串是`123456`经过 Base64 编码的结果）
```
Comment (hfxsd) [inline functions-and-operators/string-functions.md:232]:
```suggestion
以下示例解码 Base64 编码的数字 `MTIzNDU2`，该字符串是 `123456` 经过 Base64 编码的结果。
```

File: functions-and-operators/string-functions.md [hunk 14]
```diff
@@ -140,11 +140,105 @@ SELECT BIN("123q123");
...
 ### [`FORMAT()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_format)
 
-返回指定小数位数格式的数字
+`FORMAT(X,D[,locale])` 函数用于将数字X四舍五入格式化为类似于“#,###,###.##”的格式，保留D位小数，并将结果作为字符串返回。
+
+**参数：**
+
+1. **`X`：数字或数字字符串：** 要格式化的数字。可以是直接的数字值、数字字符串、或科学记数法格式的数字。
+2. **`D`：小数位数：** 指定返回值的小数位数。
+3. **`[locale]`：区域设置（可选参数）：** 可选的第三个参数允许指定一个区域设置，用于结果数字的小数点、千位分隔符和分隔符之间的分组。合法的区域设置值与 `lc_time_names` 系统变量的合法值相同。如果未指定或者区域设置为 `NULL`，则默认使用 `'en_US'` 区域设置。
```
[the remark this answers is not in the document]
Comment (ittuann) [reply functions-and-operators/string-functions.md:149]: `lc_time_names` 用于指定时间值和日期名称及缩写的本地化显示。

```sql
mysql> SHOW VARIABLES LIKE 'lc_time_names';
+---------------+-------+
| Variable_name | Value |
+---------------+-------+
| lc_time_names | en_US |
+---------------+-------+
1 row in set (0.00 sec)
```

https://dev.mysql.com/doc/refman/8.3/en/locale-support.html
https://dev.mysql.com/doc/refman/8.3/en/server-system-variables.html#sysvar_lc_time_names

[the remark this answers is not in the document]
Comment (hfxsd) [reply functions-and-operators/string-functions.md:149]: OK 我把链接加上了

File: functions-and-operators/string-functions.md [hunk 15]
```diff
@@ -140,11 +140,107 @@ SELECT BIN("123q123");
 
 ### [`FORMAT()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_format)
 
-返回指定小数位数格式的数字
+`FORMAT(X,D[,locale])` 函数用于将数字 X 四舍五入格式化为类似于 “#,###,###.##” 的格式，保留 D 位小数，并将结果作为字符串返回。
+
+该函数根据指定的小数位数进行四舍五入和格式化。如果指定的小数位数大于数字的实际小数位数，则会在结果中填充相应长度的零。
+
```
Comment (qiancai) [inline functions-and-operators/string-functions.md:146]:
```suggestion
```

File: functions-and-operators/string-functions.md [hunk 16]
```diff
@@ -140,11 +140,107 @@ SELECT BIN("123q123");
...
+#### 参数
+
+- `X`：要格式化的数字。可以是直接的数字值、数字字符串、或科学记数法格式的数字。
+- `D`：指定返回值的小数位数。
```
Comment (qiancai) [inline functions-and-operators/string-functions.md:150]:
```suggestion
- `D`：指定返回值的小数位数。该函数根据 `D` 对 `X` 进行四舍五入。如果 `D` 大于 `X` 的实际小数位数，则会在结果中填充相应长度的零。
```

File: functions-and-operators/string-functions.md [hunk 17]
```diff
@@ -140,11 +140,107 @@ SELECT BIN("123q123");
...
+- `[locale]`：指定一个区域设置，用于结果数字的小数点、千位分隔符和分隔符之间的分组。合法的区域设置值与 [`lc_time_names`](https://dev.mysql.com/doc/refman/8.3/en/server-system-variables.html#sysvar_lc_time_names) 系统变量的合法值相同。如果未指定或者区域设置为 `NULL`，则默认使用 `'en_US'` 区域设置。该参数可选。
```
Comment (qiancai) [inline functions-and-operators/string-functions.md:151]:
```suggestion
- `[locale]`：指定一个区域设置，用于结果中数字的小数点、千位分隔符和分隔符之间的分组。合法的区域设置值与 [`lc_time_names`](https://dev.mysql.com/doc/refman/8.3/en/server-system-variables.html#sysvar_lc_time_names) 系统变量的合法值相同。如果未指定或者设置为 `NULL`，则默认使用 `'en_US'` 区域设置。该参数可选。
```

File: functions-and-operators/string-functions.md [hunk 18]
```diff
@@ -140,11 +140,107 @@ SELECT BIN("123q123");
...
+
+#### 行为细节
+
+- 如果输入的第一个参数为字符串，且该字符串中只包含数字时，该函数将按照该数字返回结果。例如，`FORMAT('12.36', 1)` 与 `FORMAT(12.36, 1)` 的返回结果相同。
+- 如果输入的第一个参数为科学计数法（`E/e`）表示的数字时，该函数将按照该数字返回结果。例如，`FORMAT('1E2', 3)`），函数返回 `100.000`。
+- 如果输入的第一个参数为非数字开头的字符串时，该函数除了返回零值外，还返回一个警告 `(Code 1292)`。例如，`FORMAT('q12.36', 5)` 函数返回 `0.00000`，还会包含一个警告 `Warning (Code 1292): Truncated incorrect DOUBLE value: 'q12.36'`。
+- 如果输入的第一个参数为数字和非数字混合的字符串时，该函数将基于该参数中开头连续的数字部分返回结果，还返回一个警告 `(Code 1292)`。例如，`FORMAT('12.36q56.78', 1)` 与 `FORMAT('12.36', 1)` 的返回的数字结果相同，但 `FORMAT('12.36q56.78', 1)` 还会包含一个警告 `Warning (Code 1292): Truncated incorrect DOUBLE value: '12.36q56.78'`。
+- 如果输入的第二个参数为零，函数将四舍五入小数部分并返回整数。
+- 如果输入的第二个参数为负数，函数将四舍五入小数部分并返回整数。
```
Comment (qiancai) [inline functions-and-operators/string-functions.md:160]:
```suggestion
- 如果输入的第二个参数为零或负数，该函数将四舍五入小数部分并返回整数。
```

File: functions-and-operators/string-functions.md [hunk 19]
```diff
@@ -140,11 +140,107 @@ SELECT BIN("123q123");
...
+| FORMAT(1234.56, 1, 'en_US') |
++-----------------------------+
+| 1,234.6                     |
++-----------------------------+
+```
 
 ### [`FROM_BASE64()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_from-base64)
 
-解码 base-64 表示的字符串，并返回结果
+`FROM_BASE64(str)` 函数接收一个遵循 [Base64](https://datatracker.ietf.org/doc/html/rfc4648) 编码规则进行编码的字符串，对其解码后，将结果以十六进制字符串的形式返回。
```
Comment (qiancai) [inline functions-and-operators/string-functions.md:196]:
```suggestion
`FROM_BASE64(str)` 函数用于对 [Base64](https://datatracker.ietf.org/doc/html/rfc4648) 编码的字符串进行编码，并将解码结果以十六进制字符串的形式返回。
```

File: functions-and-operators/string-functions.md [hunk 20]
```diff
@@ -140,11 +140,107 @@ SELECT BIN("123q123");
...
+
+- 此函数接受一个单一参数，即需要解码的 Base64 编码字符串。
+- 如果输入参数为 `NULL` 或无效的 Base64 编码字符串，`FROM_BASE64()` 函数将返回 `NULL`。
+
+> **注意：**
+>
+> `FROM_BASE64()` 函数仅处理合法有效的 Base64 编码字符串。
+
```
Comment (qiancai) [inline functions-and-operators/string-functions.md:204]:
```suggestion
```

Comment (qiancai) [reply functions-and-operators/string-functions.md:204]: reason: 和前面的 “如果输入参数为 `NULL` 或无效的 Base64 编码字符串，`FROM_BASE64()` 函数将返回 `NULL`。” 意思相同

File: functions-and-operators/string-functions.md [hunk 21]
```diff
@@ -140,11 +140,107 @@ SELECT BIN("123q123");
...
+#### 示例
+
+以下示例解码 Base64 编码的字符串 `'SGVsbG8gVGlEQg=='`，该字符串是 `'Hello TiDB'` 经过 [`TO_BASE64()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_to-base64) function.) 编码的结果。
```
Comment (qiancai) [inline functions-and-operators/string-functions.md:207]:
```suggestion
以下示例解码 Base64 编码的字符串 `'SGVsbG8gVGlEQg=='`，该字符串是 `'Hello TiDB'` 经过 [`TO_BASE64()`](#to_base64) 函数编码的结果。
```

File: functions-and-operators/string-functions.md [hunk 22]
```diff
@@ -140,11 +140,107 @@ SELECT BIN("123q123");
...
+```sql
+mysql> SELECT CONVERT(FROM_BASE64('SGVsbG8gVGlEQg==') USING utf8mb4);
++--------------------------------------------------------+
+| CONVERT(FROM_BASE64('SGVsbG8gVGlEQg==') USING utf8mb4) |
++--------------------------------------------------------+
+| Hello TiDB                                             |
++--------------------------------------------------------+
+```
+
+以下示例解码 Base64 编码的数字 `MTIzNDU2`，该字符串是 `123456` 经过 [`TO_BASE64()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_to-base64) 编码的结果。
```
Comment (qiancai) [inline functions-and-operators/string-functions.md:234]:
```suggestion
以下示例解码 Base64 编码的数字 `MTIzNDU2`，该字符串是 `123456` 经过 [`TO_BASE64()`](#to_base64) 函数编码的结果。
```

File: functions-and-operators/string-functions.md (hunk 15 above)
Comment (qiancai) [reply functions-and-operators/string-functions.md:146]: reason: 看起来和参数大小有关，挪到 https://github.com/pingcap/docs-cn/pull/16119/files#r1472342805 这里来，请看看是否可以

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: functions-and-operators/string-functions.md [hunk 23]
```diff
@@ -140,11 +140,100 @@ SELECT BIN("123q123");
...
+| FORMAT(1234.56, 1, 'en_US') |
++-----------------------------+
+| 1,234.6                     |
++-----------------------------+
+```
 
 ### [`FROM_BASE64()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_from-base64)
 
-解码 base-64 表示的字符串，并返回结果
+`FROM_BASE64(str)` 函数用于对 [Base64](https://datatracker.ietf.org/doc/html/rfc4648) 编码的字符串进行编码，并将解码结果以十六进制字符串的形式返回。
```
Comment (qiancai) [inline functions-and-operators/string-functions.md:193]:
```suggestion
`FROM_BASE64(str)` 函数用于对 [Base64](https://datatracker.ietf.org/doc/html/rfc4648) 编码的字符串进行解码，并将解码结果以十六进制字符串的形式返回。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**